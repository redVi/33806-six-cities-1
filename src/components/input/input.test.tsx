import React from "react";
import renderer from "react-test-renderer";
import Input from "@/components/input/input";

describe("Input", () => {
  it("renders correctly", () => {
    const tree = renderer.create(
      <Input
        required={true}
        error="Is not valid e-mail address"
        name="email"
        placeholder="E-mail"
        label="E-mail" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
