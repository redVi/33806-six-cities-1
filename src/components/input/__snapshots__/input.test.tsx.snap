// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly 1`] = `
Array [
  <label
    className="visually-hidden"
  >
    E-mail
  </label>,
  <input
    name="email"
    placeholder="E-mail"
    required={true}
  />,
  <span
    className="error error__validation"
  >
    Is not valid e-mail address
  </span>,
]
`;
